Gemfile

```diff
@@ -75,9 +75,7 @@ end
 if ENV['GOVSPEAK_DEV']
   gem 'govspeak', path: '../govspeak'
 else
-  # 5.5.0 only because sanitize 4.6.4 is stricter on govspeak, which makes attachments invalid
-  # See https://trello.com/c/90AjvFzy/77-whitehall-isnt-able-to-use-latest-govspeak-version
-  gem 'govspeak', '~> 5.5.0'
+  gem 'govspeak', '~> 6'
```

Comment:
Does this need to be `~> 6.3` (i.e. `>=6.3` and `<7`) if we need specific features from 6.3?

Reply:
That's a good point. I'll update.